.github/release-drafter.yml

```diff
@@ -1,5 +1,6 @@
+# from  https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml
 _extends: .github
 name-template: BlueOcean $NEXT_PATCH_VERSION
 tag-template: blueocean-parent-$NEXT_PATCH_VERSION
 exclude-labels:
```

Comment:
No need, it will take it from the global config now

Reply:
no need for the exclude labels? or no need for the comment?